features/groundstate_energy.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Find the groundstate energy
```

Comment:
@horsfielda I think you'll find this interesting. I've only actually added one behave (or black box) test. In the long term we would really like to have a ton of these, to really test PyLATO properly. But for now it's a really good way to ensure that any changes we make don't inadvertently stop PyLATO from running at all (like I broke it when I merged in the python3 branch!).